app/Http/Controllers/CategoryController.php

```diff
@@ -79,6 +115,17 @@ public function update(Request $request, $id)
      */
     public function destroy($id)
     {
-        //
+        $category = Category::findOrFail($id);
+        $category->load('children');
+
+        if ($category->children->count()){
+            foreach ($category->children as $child){
+                $child->delete();
```

Comment:
```suggestion
            $category->children->delete();
```
thử như này đc ko e?

Reply:
em bị lỗi khi sửa như vậy chị ạ, em muốn nó đếm luôn cả thg con ra và xóa cha là xóa cả con luôn